packages/react/.storybook/main.ts

```diff
@@ -42,8 +39,8 @@ const config: StorybookConfig = {
     getAbsolutePath("@storybook/addon-a11y"),
     getAbsolutePath("@storybook/addon-themes"),
     getAbsolutePath("@vueless/storybook-dark-mode"),
+    // getAbsolutePath("storybook-addon-tag-badges"),
```

Comment:
Should we remove this line commented?

Reply:
Yes, that is the add-on was causing the build issue, I commented intead of removing it just in case this has side effects I didnt detect